Assets/Scripts/Player/PlayerFootsteps.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerFootsteps : MonoBehaviour
+{
+    private CharacterController controller;
+    private bool isWalking;
+    private bool keysPressed;
+
+    private void Start()
+    {
+        controller = GetComponent<CharacterController>();
+        isWalking = false;
+        keysPressed = false;
+    }
+
+    void Update()
+    {
+        HandleFootsteps();
+        HandleAudio();
+    }
+
+    private void HandleFootsteps()
+    {
+        keysPressed = Input.GetKey(KeyCode.W) || Input.GetKey(KeyCode.A) || Input.GetKey(KeyCode.S) || Input.GetKey(KeyCode.D);
+    }
```

Comment:
This is not using the new input system, and as a result is not consistent with the rest of the input code. Access the input from `HuntingInputManager` instead